builder/src/scss/theme/_bootswatch.scss

```diff
@@ -15,6 +15,10 @@ $web-font-path: "https://fonts.googleapis.com/css?family=Lato:400,700,400italic"
   }
 }
 
+.btn-secondary:hover {
```

Comment:
What's this change for? I don't see any reference with `.btn-secondary` in the diff, so just wondering

Reply:
Modifies the color on hover of top rated/most downloaded/etc sort buttons to match that of the navbar and dropdown.

Reply:
Hmmm, the color scheme comes from the bootstrap theme, and if it's edited I think that's the proper place to do it. There's some SCSS variables you can set for the theme if you want to change something.

Personally I feel like a color change isn't needed for those buttons since the backround is already changing.